lighty-applications/lighty-rcgnmi-app-aggregator/lighty-rcgnmi-app/src/main/java/io/lighty/applications/rcgnmi/app/RCgNMIApp.java

```diff
@@ -30,21 +31,28 @@
 import org.slf4j.LoggerFactory;
 
 public class RCgNMIApp {
-    private static final Logger LOG = LoggerFactory.getLogger(RCgNMIApp.class);
 
-    private static final String UNABLE_TO_START_APPLICATION = "Unable to start lighty.io application!";
+    private static final Logger LOG = LoggerFactory.getLogger(RCgNMIApp.class);
+    private static final String UNABLE_TO_START_APPLICATION = "Unable to start RCgNMI lighty.io application!";
+    private static final String UNABLE_TO_STOP_APPLICATION
+            = "Exception was thrown while shutting down RCgNMI lighty.io application!";
 
     private AbstractLightyModule rcgnmiLightyModule;
 
     // Using args is safe as we need only a configuration file location here
     @SuppressWarnings("squid:S4823")
     public static void main(final String[] args) {
         final RCgNMIApp app = new RCgNMIApp();
-        app.start(args);
+        try {
+            app.start(args);
+        } catch (RcGnmiAppException e) {
+            LOG.error(UNABLE_TO_START_APPLICATION, e);
+            System.exit(0);
```

Comment:
Don't use `System.exit(0)`. If shutdown procedure is done correctly, application should shutdown itself.